docs/scripts/reference.py

```diff
@@ -593,30 +593,19 @@ def _format_tag_signature(tag_spec: TagSpec) -> str:
     {% endcomponent %}
     ```
     """
-    params: List[str] = [tag_spec.tag]
-
-    if tag_spec.positional_only_args:
-        params.extend([*tag_spec.positional_only_args, "/"])
-
-    optional_kwargs = set(tag_spec.optional_kwargs or [])
-
-    params.extend([f"{name}=None" if name in optional_kwargs else name for name in tag_spec.pos_or_keyword_args or []])
-
-    if tag_spec.positional_args_allow_extra:
-        params.append("[arg, ...]")
-
-    if tag_spec.keywordonly_args is True:
-        params.append("**kwargs")
-    elif tag_spec.keywordonly_args:
-        params.extend(
-            [f"{name}=None" if name in optional_kwargs else name for name in (tag_spec.keywordonly_args or [])]
-        )
+    # The signature returns a string like:
```

Comment:
This updates the logic how we generate the API reference for template tags (e.g. `{% component  %}`, etc).